templates/zerver/api/get-single-user.md

```diff
@@ -0,0 +1,56 @@
+# Get a user
+
+Retrieve a single user in a realm.
+
+`GET {{ api_url }}/v1/users/{user_id}`
+
+## Usage examples
+
+{start_tabs}
+{tab|python}
+
+{generate_code_example(python)|/users/{user_id}:get|example}
+
+{tab|curl}
+
+{generate_code_example(curl, include=[""])|/users/{user_id}:get|example}
+
+You may pass the `client_gravatar` or `include_custom_profile_fields` query parameter as follows:
+
+{generate_code_example(curl)|/users/{user_id}:get|example}
+
+{end_tabs}
+
+## Arguments
+
+**Note**: The following arguments are all URL query parameters.
+
+{generate_api_arguments_table|zulip.yaml|/users/{user_id}:get}
+
+## Response
+
+#### Return values
+
+* `members`: A list with a single dictionary that contains information
+  about a particular user or bot.
+    * `email`: The email address of the user or bot..
```

Comment:
Copyediting is needed here; this shouldn't end with `..`.

Reply:
Also, now that I see how this looks in the documentation, maybe we want to change this endpoint to return the user in a more intuitive form, not as a single-item list?  Maybe `user: {}` rather than `members: []`?

Reply:
@timabbott I'll add a commit for this in the PR I've proposed below.